### ISSUE TITLE
Force the usage of named argument

### ISSUE THREAD

**Issue by PHELAT** [arturbosch/detekt#2151]
## Expected Behavior of the rule
Being able to force named argument usage:
```kotlin
// Won't compile
myInstance.doSomeThing("Hello World")
```
```kotlin
// Will compile
myInstance.doSomeThing(message = "Hello World")
```

## Context
To increase code readability and easier code maintenance, I would like to have the ability to force the usage of named arguments.

**Comment by schalkms** [comment]
I think this is an issue of making bad code more readable, rather than **best practice**. When functions have one to three parameters it should be clear from the method name what the parameter mean. The named parameter add nothing to the table. This is rather the ideal case to be in. These days IDEA does a good job by displaying inline parameter name hints if the parameter is not obvious from the function call. You can even display these hints on all function invocations in IntelliJ.

Please note, that detekt doesn't prevent `myInstance.doSomeThing("Hello World")` from compiling. Detekt is a static code analyzer and only flags this code snippet. Since your mentioned code style is rather special and because of the outlined arguments, I'd encourage you to write a custom domain-specific detekt rule [1], which flags this code, if you really want to enforce it.
Detekt tries to include rules that are useful for the majority of the user base.

[1] https://arturbosch.github.io/detekt/extensions.html#custom-rulesets
